source/linux/basic/usersystem.rst

```diff
@@ -0,0 +1,82 @@
+Usersytem Basics
+================
+
+Linux has a multiuser user system which allows multiple users to work on one system at the same time. 
+Linux' user system includes a permission system (more on that here: :ref:`permissions` ).
+The following commands will give you a basic overview:
```

Comment:
Same with this chapter here. This is actually out of scope. The customer might even think this is how this is supposed to be handled on the target. I am not sure if this is not too confusing.

Reply:
I've added a note and moved this chapter to the users and permissions chapter. 
It would be good if you could take a quick look at it.